docs: Update OpenTelemetry instrumentation docs for Express, JavaScript, NestJS, and Next.js

animeshd9 opened the pull request:
- Added TypeScript and JavaScript code examples for tracing configuration
- Improved documentation clarity and added more detailed instructions

Fixes: https://github.com/SigNoz/signoz-web/issues/95

paulohmorais commented:
The link of json on tutorial need approve of this PR too

https://github.com/SigNoz/dashboards/pull/18

paulohmorais commented:
Updated doc, with metrics and logs now. 

The link of dashboard on tutorial keeps on this https://github.com/SigNoz/dashboards/pull/18

ankit01-oss commented:
@Calm-Rock please take up the review for this.

Calm-Rock commented:
Hey @paulohmorais - we have migrated to a new website - https://github.com/SigNoz/signoz-web

Can you please raise the PR on the new website.

animeshd9 committed:
docs: Update OpenTelemetry instrumentation docs for Express, JavaScript, NestJS, and Next.js

- Added TypeScript and JavaScript code examples for tracing configuration
- Updated region-specific endpoints for SigNoz Cloud
- Improved documentation clarity and added more detailed instructions
- Uncommented previously hidden sections and added missing content
- Enhanced code snippets with better explanations and environment variable handling

animeshd9 committed:
chore: Reverted few changes

animeshd9 committed:
Merge branch 'ts-docs' of github.com:animeshd9/signoz-web into ts-docs
